roles/idrac_bios/molecule/update_attributes_immediate_with_jobwait/prepare.yml

```diff
@@ -0,0 +1,29 @@
+---
```

Comment:
if prepare.yml is common for all the scenario, have a look to molecule documentation, we can have common file throughout the scenario.

https://ansible.readthedocs.io/projects/molecule/configuration/#molecule.provisioner.ansible.Ansible

![image](https://github.com/dell/dellemc-openmanage-ansible-modules/assets/110585390/10ea87da-19eb-420d-a833-0fd7780c0e65)

Reply:
Updated. Now, a single file for prepare and cleanup are there.